Lib/ntpath.py

```diff
@@ -334,13 +334,13 @@ def isreserved(path):
     """Return true if the pathname is reserved by the system."""
     # Refer to "Naming Files, Paths, and Namespaces":
     # https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file
-    path = os.fsdecode(splitroot(path)[2]).replace(altsep, sep)
-    return any(_isreservedname(name) for name in reversed(path.split(sep)))
+    path = os.fsdecode(splitroot(path)[2]).replace('/', '\\')
```

Comment:
Must I move this to a different pull request? It's probably possible to break the old code by monkey patching os.path.altsep or os.path.sep.

Reply:
It's fine to stay in this PR! Those sorts of implementation details aren't usually covered by the API contract with users, and many other functions in this module define their own local `sep` / `altsep` variables.